src/params.rs

```diff
@@ -22,13 +22,11 @@ pub struct Params {
 impl Params {
     pub fn get_minsize(&self) -> Option<u64> {
         match &self.minsize {
-            Some(msize) => {
-                match msize.parse::<bytesize::ByteSize>() {
-                    Ok(units) => Some(units.0),
-                    Err(_) => None
-                }
+            Some(msize) => match msize.parse::<bytesize::ByteSize>() {
+                Ok(units) => Some(units.0),
+                Err(_) => None,
             },
-            None => None
+            None => None,
```

Comment:
This is rustfmt doing its thing

Reply:
I realize its maybe because I have a 120 chars limit on line length, let me know if that's not in line with this repo's style guidelines